<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyProgramming</class>
 <widget class="QDialog" name="ModifyProgramming">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Of Input Files: </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numOutput">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Environment:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numInput">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_activityId">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>activityid</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_numTests">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_environment">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>c++</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Java</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Eclipse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QT</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activity ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_testID_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>testID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update_P_Test">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Programming Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Test:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_testID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*testID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Number Of Output Files:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
